src/Aspire.Hosting/DistributedApplicationResourceBuilder.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Aspire.Hosting;
 
-internal sealed class DistributedApplicationResourceBuilder<T>(IDistributedApplicationBuilder applicationBuilder, T resource) : IResourceBuilder<T> where T : IResource
+public sealed class DistributedApplicationResourceBuilder<T>(IDistributedApplicationBuilder applicationBuilder, T resource) : IResourceBuilder<T> where T : IResource
```

Comment:
Nice! Should we make it easier to make a builder from a resource? Extension method?

Reply:
Yes. I was thinking the same thing.

Reply:
Done.